SROS2_Linux.md

```diff
@@ -120,24 +120,27 @@ To be able to use the ros2 CLI tools to interact with your secured system, you n
 export ROS_SECURITY_ENCLAVE_OVERRIDE=/talker_listener/listener
 ```
 
-Then use the CLI as usual:
+Then use the CLI with `--no-daemon` and `--spin-time`:
+
+> [!NOTE]
+> Avoid using `ros2 daemon` since it may not have security enclaves, and enough time duration should be given for the discovery in secured network.
 
 ```bash
-ros2 node list
+ros2 node list --no-daemon --spin-time 3
```

Comment:
3 seconds come from my amd64 environment, it should give it enough time window for the security handshake and discovery. (actually, fast-dds security takes much longer time compared to cyclonedds, that is why we did not see this issue with cyclonedds. but possibly this is common issue.)